src/NuGet.Core/NuGet.Protocol.Core.v3/HttpSource/HttpRetryHandler.cs

```diff
@@ -2,17 +2,21 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Diagnostics;
 using System.Globalization;
 using System.Net.Http;
 using System.Threading;
 using System.Threading.Tasks;
-using NuGet.Common;
+using NuGet.Logging;
 using NuGet.Protocol.Core.v3;
 
 namespace NuGet.Protocol
 {
     public class HttpRetryHandler
     {
+        public const string RequestLogFormat = "  {0} {1}";
+        public static readonly string ResponseLogFormat = "  {0} {1} {2}" + Strings.Milliseconds;
```

Comment:
Do you know that is true for all languages? E.g. in the use you say $5 for currency but elsewhere 5$. It won't be confusing if we have a comment on the resx file